### PR TITLE
Refactor the TAG docs based on repeated questions (#1923)

### PR DESCRIPTION
**Pull request by emmanuelkeller**
(Backport branch 2.0)

* Refactor the TAG docs based on repeated questions

* Fix an FT.CREATE example

* Fix typo

* Fix a couple of typos

### CODE REVIEW

File: docs/Tags.md [hunk 1]
```diff
@@ -73,19 +81,58 @@ FT.SEARCH myIndex "@cities:{ New York | Los Angeles | Barcelona }"
...
 
-@tags:{foo bar baz | hello world }
+You can see what that looks like in the following example:
+
+```
+127.0.0.1:6379> FT.SEARCH idx "@tags:{ to be or not to be }"
+(error) Syntax error at offset 27 near be
 ```
+
+**NOTE:** Stop words are words that are so common that a search engine ignores them. We have a dedicated page about [stop words in RediSearch](https://oss.redislabs.com/redisearch/Stopwords/#default_stop-word_list) if you would like to learn more.
```
Comment (ashtul) [inline docs/Tags.md:136]: I believe we have changed this and tag fields do index stopword.
See PR #1668

**Review by ashtul — COMMENTED** [review]
Looks good. One comment about stopwords.

File: docs/Tags.md (hunk 1 above)
Comment (abrookins) [reply docs/Tags.md:136]: @ashtul What version included that change? I'm on 2.0.5 and still see the behavior in this document. E.g. (in this example, author_ids is a TAG field):

```
127.0.0.1:6379> FT.SEARCH books-idx "@author_ids:{ to be }"
(error) Syntax error at offset 20 near be
127.0.0.1:6379> FT.SEARCH books-idx "@author_ids:{ to\\ be }"
1) (integer) 0
```

CC @emmanuelkeller

Comment (emmanuelkeller) [reply docs/Tags.md:136]: @abrookins It is fixed in 2.0.6
https://github.com/RediSearch/RediSearch/releases/tag/v2.0.6

Comment (abrookins) [reply docs/Tags.md:136]: Still appears to be an issue in 2.0.6:

```
1:M 22 Apr 2021 17:24:56.508 * <search> RediSearch version 2.0.6 (Git=v2.0.4-55-gb0db8b2e)
...
127.0.0.1:6379> FT.SEARCH books-idx "@author_ids:{ to be }"                                                                                             (error) Syntax error at offset 20 near be
```

**Pull request closed without merging**